content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/accordion.html

```diff
@@ -17,13 +17,13 @@
      id="${accordion.id}"
      class="cmp-accordion"
      data-cmp-is="accordion"
-     data-cmp-data-layer="${accordion.dataLayer.string}"
+     data-cmp-data-layer="${accordion.componentDataModel.string}"
```

Comment:
With the changes that I suggested, it would become:
`accordion.componentData.json` (or `accordion.componentData.jsonString`)
Even better and more concise notation would be:
`accordion.data.json`

Reply:
Done in 2d1846a, now we use `component.data.json`